### PR TITLE
NXP-26450: nuxeo-results-view element documentation

### PR DESCRIPTION
**Pull request by gcarlin**

### CODE REVIEW

File: elements/search/nuxeo-results-view.html [hunk 1]
```diff
@@ -226,7 +228,7 @@
           value: false
         },
         /**
-         * If `true`, automatically execute the search each time a param is changed.
+         * If true, automatically execute the operation when either `provider` or `params` changes.
```
Comment (guirenard) [inline elements/search/nuxeo-results-view.html]: not sure `operation` is correct, we are performing a search here

**Comment by guirenard** [comment]
:+1:

File: elements/search/nuxeo-results-view.html (hunk 1 above)
Comment (Gabez0r) [reply elements/search/nuxeo-results-view.html]: agree, this should be changed

**Comment by Gabez0r** [comment]
👍

**Pull request merged**